products/kirby.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Kirby
+category: server-app
+iconSlug: kirby
+permalink: /kirby
+alternate_urls: 
+-   /getkirby
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: true
+releaseLabel: "__RELEASE_CYCLE__{%if r.codename %} ({{r.codename}}){%endif%}"
```

Comment:
We don't support using the release as a variable. Working on supporting something like this in a branch, but till then, we have a workaround:

```suggestion
releaseLabel: "__RELEASE_CYCLE__{%if '__CODENAME__' %} (__CODENAME__){%endif%}"
```

Reply:
@captn3m0 this one is exactly used under mongodb product. so we need to fix that too

Reply:
Yes, had noticed. Will take it up.

Reply:
Thanks for adding this. I was trying to achieve the same, but could not make it work with Liquid 👍